script.module.t1mlib/addon.xml

```diff
@@ -13,7 +13,7 @@
         <email></email>
         <website></website>
         <forum>http://forum.kodi.tv/showthread.php?tid=247163</forum>
-        <source>https://github.com/learningit/repository.t1m.v2.addons/tree/master/src/script.module.t1mlib</source>
+        <source>https://github.com/xbmc/repo-scripts/tree/isengard/script.module.t1mlib</source>
```

Comment:
Reasonable. The issue is that I am no longer maintaining the sources in the individual repos as I have way too many add-ons to keep messing with multiple versions. I do all the work against what is in the official repo, so the ones I have are not up to date. I am having serious issues with people loading the add-ons from anywhere and it gets more confusing under Jarvis because users have no idea which repo they should update against. It's a real mess.